src/webapp/features/rabble/board/Square.tsx

```diff
@@ -20,10 +21,29 @@ const Square = ({ square, clickSquare, playableLocations }: SquareProps) => {
     <div className={styles.displayBonus}>{square.bonus}</div>
   );
 
+  let playableClass = "";
+  if (selectedLocation && playableLocations.includes(location)) {
```

Comment:
Ah, here it is - doesn't work for selectedLocation 0.  Easy fix.

Reply:
Cool - now I'm just making sure it's not null